Main

seongtaehong opened the pull request.

seongtaehong committed:
Main

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/EleutherAI/lm-evaluation-harness?pullRequest=3250) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/EleutherAI/lm-evaluation-harness?pullRequest=3250) before we can accept your contribution.<br/>**0** out of **4** committers have signed the CLA.<br/><br/>:x: seongtaehong<br/>:x: junkim100<br/>:x: sengtaeHong<br/>:x: cyc9805<br/><hr/>**sengtaeHong** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/EleutherAI/lm-evaluation-harness?pullRequest=3250) it.</sub>

seongtaehong reviewed (commented):
Add click benchmark

seongtaehong reviewed (commented):
Add click benchmark

junkim100 committed:
Implemented kobalt

cyc9805 committed:
Implemented gsm8k_symbolic

seongtaehong committed:
add mmlu_redux

junkim100 committed:
Fix dataset configuration in kobalt.yaml by removing test_split

junkim100 committed:
Merge pull request #1 from EleutherAI/main

seongtaehong committed:
add babilong benchmark

cyc9805 committed:
Update GSM8K-symbolic

seongtaehong committed:
add zebralogic benchmark

cyc9805 committed:
Add M-IFEval

cyc9805 committed:
Merge branch 'main' of https://github.com/junkim100/lm-evaluation-harness

cyc9805 committed:
Fix Kobalt loading issue

cyc9805 committed:
Fix Kobalt configuration

junkim100 committed:
fix(kobalt): handle capitalized Question/Answer fields in process_docs

- Accept both lowercase and capitalized keys from KoBALT-700: question/Question and answer/Answer
- Prevents zero-doc issue leading to “task.build_requests() did not find any docs” on the kobalt_v1 raw split

junkim100 committed:
Update pyproject.toml

cyc9805 committed:
update: implemented  math-verify to hrm

cyc9805 committed:
Merge branch 'main' of https://github.com/junkim100/lm-evaluation-harness

cyc9805 committed:
fix: aggregation issue of hrm8k

Commit:
feat(babilong): add boxed guidance to answer formatting

- Introduced BOXED_GUIDANCE to standardize answer presentation.
- Updated multiple doc_to_text functions to include boxed guidance in prompts.
- Modified regex in the default template to accommodate boxed answers.

Commit:
babilong update
